spec/javascripts/Settings/components/Common/__snapshots__/FormCollection.spec.jsx.snap

```diff
@@ -0,0 +1,31 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`should render correctly 1`] = `
+<FormFieldset
+  id="fieldset-SomeGoodBands"
+>
+  <FormLegend>
+    Some Good Bands
+  </FormLegend>
+  <GoodBand
+    key="0"
+    name="The Rolling Stones"
```

Comment:
```suggestion
    name="Justin Bieber"
```

Reply:
Can we please keep it The Rolling Stones?